pkgs/by-name/ya/yafc-ce/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  buildDotnetModule,
+  lib,
+  fetchFromGitHub,
+  dotnetCorePackages,
+  SDL2,
+  SDL2_image,
+  SDL2_ttf,
+}:
+let
+  dotnet = dotnetCorePackages.dotnet_8;
+  pname = "yafc-ce";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "shpaass";
+    repo = "yafc-ce";
+    rev = version;
+    hash = "sha256-qJOEdP2jRqK0E9Q4n7YgO5jyHxztb8aVOTlLDq1VvMk=";
+  };
+in
+buildDotnetModule {
```

Comment:
Can you please use `finalAttrs` here so that  `pname`, `version`, `src` are not in the let-in block and `meta` has no `rec` like so:

```nix
let
  dotnet = dotnetCorePackages.dotnet_8;
in
buildDotnetModule (finalAttrs: {
  pname = "yafc-ce";
  version = "2.0.1";

  src = fetchFromGitHub {
    owner = "shpaass";
    repo = "yafc-ce";
    rev = finalAttrs.version;
    hash = "sha256-qJOEdP2jRqK0E9Q4n7YgO5jyHxztb8aVOTlLDq1VvMk=";
  };
```